src/options.ts

```diff
@@ -15,11 +15,7 @@ joycon.addLoader({
 export const getOptions = (
   cwd: string,
 ): { jsxFactory?: string; jsxFragment?: string; target?: string } => {
-  let { data, path } = joycon.loadSync(["tsconfig.json"], cwd);
-
-  if (!path) {
-    ({ data, path } = joycon.loadSync(["jsconfig.json"], cwd));
-  }
+  const { data, path } = joycon.loadSync(["tsconfig.json", "jsconfig.json"], cwd);
```

Comment:
ah ok, I didn't know it was possible to pass multiple filenames, my bad :)

thank you